sound/soc/sof/intel/hda-stream.c

```diff
@@ -433,15 +433,23 @@ int hda_dsp_stream_hw_params(struct snd_sof_dev *sdev,
 irqreturn_t hda_dsp_stream_interrupt(int irq, void *context)
```

Comment:
fix your commit message

ASoC: SOF: Intel: hda-stream: handle real stream interrupts only

The stream and IPC share the same interrupt. The stream interrupt handler mistakenly uses the ipc interrupt
and return IRQ_HANDLED, causing the ipc interrupt to be missed.

Make sure the stream interrupt handler only deals with stream-related interrupts.

Reply:
Good, thanks, let me take it.